### ISSUE TITLE
Symptoms radiating into arms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10133]
**Preferred term label:**
Symptoms radiate into right arm
Symptoms radiate into left arm

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1960990
https://www.ncbi.nlm.nih.gov/medgen/C1960981

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Myocardial infarction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Shouldn't this be
Pain radiating to right arm (rather than "symptoms")?

**Comment by MickeySegal** [comment]
We elaborate symptoms as "pain or other discomfort".

**Comment by pnrobinson** [comment]
@MickeySegal 
adding as
Adding new term: 
- ID: HP:6000132
- label: Radiation into left arm
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Refers to pain or discomfort that is perceived to travel from the chest into the left arm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10133
- synonyms: Symptoms radiate into left arm(NONE)


AND

Adding new term: 
- ID: HP:6000133
- label: Radiation into right arm
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Refers to pain or discomfort that is perceived to travel from the chest into the right arm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10133
- synonyms: Symptoms radiate into right arm(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 

sorry, there was an error, the correct term ids are

HP:0430044  Radiation into left arm
HP:0430045 Radiation into right arm
